packages/components/src/index.ts

```diff
@@ -20,7 +20,7 @@ export * from './components/Dropdown/Dropdown';
 export * from './components/form/Input/Input';
 export * from './components/form/Textarea/Textarea';
 export * from './components/form/Select/Select';
-export * from './components/form/Checkbox/Checkbox';
+export { Checkbox, type CheckboxProps } from './components/form/Checkbox/Checkbox';
```

Comment:
Why is this the only not exported like the others?

Reply:
Because of the `Multiple exports of name 'Label'` error. I wanted to deal with it after I merge `Input` since the issue is because of it, just rename what needs to be renamed.